alpha-lab/gpt4-eyes/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Assistive scene understanding with GPT-4 vision"
+description: "Exploring the use of OpenAI's GPT-4V and real-time APIs from Neon or Invisible to assist individuals with low vision or communication disabilities."
+permalink: /alpha-lab/gpt-4-eyes/
+meta:
+  - name: twitter:card
+    content: summary
+  - name: twitter:image
+    content: "https://example.com/article_image.jpg"
+  - name: twitter:player
+    content: "https://www.youtube.com/embed/_k5QbpC84H8"
+  - name: twitter:width
+    content: "1280"
+  - name: twitter:height
+    content: "720"
+  - property: og:image
+    content: "https://i.ytimg.com/vi/_k5QbpC84H8/maxresdefault.jpg"
+tags: [Pupil Invisible, Neon]
+---
+<script setup>
+import TagLinks from '@components/TagLinks.vue'
+</script>
+
+# Assistive Scene Understanding With GPT-4V
+
+<TagLinks :tags="$frontmatter.tags" />
+
+<Youtube src="_k5QbpC84H8"/>
+
+**🔉 Play the video above with audio!**
+
+::: tip
+Imagine an AI assistant that not only talks but also *“sees”* for you. Welcome to the era of Large Multimodal Models! 
+:::
+
+## Combining Eye Tracking With AI for Assistive Applications
+
+Unlike popular Large *Language* Models (LLMs), which can only interpret and interact with text, Large *Multimodal* Models (LMMs) are also able to understand images. This means chatbots, like GPT, can understand, describe and interpret the content of real-world scenes, including identifying different objects and how people might interact with them.
+
+When combined with eye tracking, this holds potential for assistive technologies. For example, it could help individuals with low vision to identify the objects at which they are gazing, highlight potential hazards in their surroundings, or assist people with ALS who face significant communication challenges.
+
+Here, we show you how to build a simple, explorative, gaze-based assistive tool using Neon or Pupil Invisible and a new LMM, GPT-4V.
+
+## What Tools Enable This
+
+OpenAI recently unveiled a set of new APIs at their [DevDay event,](https://openai.com/blog/new-models-and-developer-products-announced-at-devday) introducing GPT-4V, an extension of their most advanced large language model capable of [understanding images](https://platform.openai.com/docs/guides/vision). 
+We got access to the preview and decided to see what is capable of when integrated with our eye trackers. We used Neon and Pupil Invisible because they remove the barrier of calibration, making them more suitable for real-life applications, longer recording sessions, and thus use alongside GPT-4V for experimenting with assistive applications.
```

Comment:
Was PI used here?

Reply:
> and thus use alongside GPT-4V for experimenting with assistive applications.

Can you improve this sentence especially the last part quoted above?

Reply:
> Was PI used here?

No, but it would work with the same code, out of the box.

> Can you improve this sentence, especially the last part quoted above?
What about?

We gained access to the preview and decided to see what their model is capable of when integrated with our eye trackers. We used Neon and Pupil Invisible since they remove the barrier of calibration, making them more suitable for longer sessions and real-life scenarios and, therefore, in a better position to experiment with assistive applications.

Reply:
We gained access to the preview and decided to see what their model is capable of when integrated with our eye trackers. We used Neon and Pupil Invisible since they remove the barrier of calibration, making them more suitable for longer sessions and real-life scenarios and, therefore, in a good position to experiment with assistive applications